docs/_documentations/remotedeploy-projects.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+title: Projects in the cloud
+description: Projects in the cloud
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects
+type: document
+---
+
+# Objectives
+Now that you have configured Codewind to be used remotely, you are now ready to:
+
+1. create a new project
+2. move a project from your local Codewind instance to your remote Codewind instance on the cloud
```

Comment:
Initial caps

Reply:
note this topic renamed to 'remotedeploy-projects-vscode.md